content/authorization.md

```diff
@@ -74,20 +74,41 @@ register applications. Obtaining a Cerner Care account is free and
 requires nothing but an email address.
 
 If registering an application that is using the public app profile and requires access only when the user is online, 
-your client application must first register using our [code console][CERNER-CODE-CONSOLE]. 
+your client application must first register using our [code Console][CERNER-CODE-CONSOLE]. 
 
 If registering a confidential client application (required for offline_access and
-access on behalf of a system), please first request an account via the **non-production** instance of 
-[Cerner Central System Account Registration][SYSTEM-ACCOUNT-REGISTRATION]. Once the request is completed, use the
-issued system account GUID when registering the application in the [code console][CERNER-CODE-CONSOLE]. Be aware that
-access on behalf of a system currently has limited availability. Check the respective FHIR<sup>®</sup> implementation
-documentation to determine availability.
+access on behalf of a system), follow the directions in the [Registering a System Account][SYSTEM-ACCOUNT-SECTION] 
+section. Check the respective FHIR<sup>®</sup> implementation documentation to determine availability.
 
 Once registered, the **client identifier** will be provided for use with 
 the Cerner Authorization Server. As a registered client, Cerner 
 organizations may then ask for your client application to be enabled, 
 which is necessary in order to gain access to their protected resources.
 
+### Registering a System Account ###
+
+If an application will be using the offline_access scope, or accessing data on behalf of a system, it will need to
+maintain a secret. Currently, our implementation provides management and rotation functionality for this workflow in our 
+System Accounts application. In order to register one of these applications, you must first request a system account,
+and then register that as a SMART or FHIR application in our code Console.
```

Comment:
Does this intentionally read `SMART *or* FHIR`?

Reply:
Yes, because many people don't consider "access on behalf of a system" a SMART app, and we have apps that use SMART but don't use FHIR as well (though those generally wouldn't need offline_access).